README.md

```diff
@@ -33,10 +33,11 @@ Running pipelines on ci.cloud.gov is identical to running pipelines locally, wit
 fly -t cloud login —c https://ci.cloud.gov
 cp config/prod.example.yml config/prod.yml
 # modify config/prod.yml
-fly set-pipeline -t cloud -n -c pipelines/zap.yml -p zap --load-vars-from config/prod.yml
+# build the ZAP pipeline
```

Comment:
Any reason not to include the command or point to the directions in the zap readme here?